### ISSUE TITLE
"Position aircraft in air" bug

### ISSUE THREAD

**Issue by wlbragg** [Juanvvc/c172p-detailed#363]
Using "Position aircraft in air" causes the pontoons gears, (at the least), to function incorrectly from that point forward. It appears the bogey point are no longer valid?

**Comment by gilbertohasnofb** [comment]
I also noticed that before. But the "position aircraft in the air" seems to cause problems with almost every aircraft in FG (it's almost unusable in my opinion).

**Comment by dany93** [comment]
At start, JSBSim does iterations to position the aircraft on the ground taking into account its weight and the spring constants. We sometimes see them in the terminal.
I think it misses this step when you start in the air.

**Comment by gilbertohasnofb** [comment]
I think that if we will try to fix this, we should have a look on other bugs related to position aircraft in the air as well. For instance, if you select this option with the engine running, it will be off once you are in the air and the plane won't be so different from a stone :wink:

**Comment by dany93** [comment]
For the gears when starting in the air, I'm pessimistic (cf. the iterations lacking).
Are they really bugs? Or merely a start in the air with initial parameters and zero speed?

**Comment by wlbragg** [comment]
I agree this may be a non-starter. But I was testing and using it for convenience and noticed it was not initializing the gear at all. Even if I land and retract and extend them they act like the contacts are not there. I did not look at the props to see the condition of the contact.
I think we should at least take a look at this briefly and see if it is possible to get the gears working after starting in this mode. If it is and not too much effort then I agree with gilbertohasnofb that we might want to see if we can also start the engine.

However dany93 is probably going to be correct in his assessment of the situation, it has always been buggy at best with all aircraft.

**Comment by dany93** [comment]
If you wish to be quickly in the air, which works is to start on the ground or the water, take off to get some initial speed, and do Ctrl-u several times.
Not so convenient, I admit.

**Comment by gilbertohasnofb** [comment]
@onox @wlbragg @dany93 Is this something we can realistically tackle on our side or is this part of FG's buggy reset and position in the air features that don't work very well?

**Comment by wlbragg** [comment]
I would need to test it now and see where we are at. There has been some recent work done to the in air reset. So it is quite possible the in air reset works better already.
I'll test the state of the system as it currently stands.

**Comment by wlbragg** [comment]
By the way this goes hand-in-hand with the states work and maybe should be addressed then

**Comment by dany93** [comment]
>  Is this something we can realistically tackle on our side... ?

As far as for the gear functioning, I've not changed my mind since[ this message](https://github.com/c172p-team/c172p/issues/363#issuecomment-118097079).
